tests/core/test_model.py

```diff
@@ -1231,23 +1233,23 @@ def test_star_expansion(assert_exp_eq) -> None:
 
     assert_exp_eq(
         context.models["db.model2"].render_query(snapshots=snapshots),
-        """
+        f"""
         SELECT
           "model1"."id" AS "id",
           "model1"."item_id" AS "item_id",
           "model1"."ds" AS "ds"
-        FROM "sqlmesh__db"."db__model1__3784582009" AS "model1"
+        FROM "sqlmesh__db"."db__model1__{snapshots['db.model1'].version}" AS "model1"
```

Comment:
Did fingerprint change here because of your changes? Why? These models don't have clustered_by

Reply:
local sqlglot version triggered this, i decided to change this for future stuff